### ISSUE TITLE
Bad error message when project attach fails b/c package folder does not exist

### ISSUE THREAD

**Issue by klazuka** [klazuka/intellij-elm#158]
The problem is that `File#listFiles` returns null when the directory does not exist.

<img width="982" alt="screen shot 2018-12-09 at 10 17 24 am" src="https://user-images.githubusercontent.com/84525/49701051-b16ccb80-fb9b-11e8-90a6-2f698d94d617.png">

```
2018-12-09 10:17:05,182 [169615971]  ERROR - .workspace.ElmWorkspaceService - Unexpected error when loading /Users/keith/dev/elm/elm-dev-tools/elm.json 
java.lang.IllegalStateException: File("$elmHomePath/$comp…             .listFiles() must not be null
	at org.elm.workspace.ElmToolchain.availableVersionsForPackage(ElmToolchain.kt:95)
	at org.elm.workspace.ElmProjectKt.constraintDepsToPackages(ElmProject.kt:236)
	at org.elm.workspace.ElmProjectKt.access$constraintDepsToPackages(ElmProject.kt:1)
	at org.elm.workspace.ElmProject$Companion.parse(ElmProject.kt:147)
	at org.elm.workspace.ElmProject$Companion.parse(ElmProject.kt:96)
	at org.elm.workspace.ElmProject$Companion.parse$default(ElmProject.kt:93)
	at org.elm.workspace.ElmWorkspaceService$asyncLoadProject$1.invoke(ElmWorkspaceService.kt:167)
	at org.elm.workspace.ElmWorkspaceService$asyncLoadProject$1.invoke(ElmWorkspaceService.kt:59)
```

**Comment by klazuka** [comment]
Also, the "Could not load $package" error message should:
- ask whether the user is sure that the package is installed
- show a nice `toString` for version constraints

And make sure that this is covered by a test.

**Comment by klazuka** [comment]
This was fixed by 595f1a8
